soc/nordic/common/vpr/Kconfig.sysbuild

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig VPR_LAUNCHER
+	bool "VPR launcher"
+	depends on SOC_SERIES_NRF54HX
+	help
```

Comment:
would also add another depends on here, assuming that people will want to use this option when building for the viper core, so `depends on SOC_NRF54H20_CPUPPR` would work, otherwise someone could build this for the application core with another image which will create 2 images overlapping the same address space